apps/application-system/api/src/environments/environment.ts

```diff
@@ -34,84 +48,98 @@ const devConfig = {
     smsOptions: {
       url: 'https://smsapi.devnova.is',
       username: 'IslandIs_User_Development',
-      password: process.env.NOVA_PASSWORD,
+      password: process.env.NOVA_PASSWORD ?? '',
       acceptUnauthorized: true,
     },
-    presignBucket: process.env.FILE_SERVICE_PRESIGN_BUCKET,
-    attachmentBucket: process.env.APPLICATION_ATTACHMENT_BUCKET,
+    presignBucket: process.env.FILE_SERVICE_PRESIGN_BUCKET ?? '',
+    attachmentBucket: process.env.APPLICATION_ATTACHMENT_BUCKET ?? '',
     generalPetition: {
       endorsementsApiBasePath: 'http://localhost:4246',
     },
     userProfile: {
       serviceBasePath: 'http://localhost:3366',
     },
     islykill: {
-      cert: process.env.ISLYKILL_CERT,
-      passphrase: process.env.ISLYKILL_SERVICE_PASSPHRASE,
-      basePath: process.env.ISLYKILL_SERVICE_BASEPATH,
+      cert: process.env.ISLYKILL_CERT ?? '',
+      passphrase: process.env.ISLYKILL_SERVICE_PASSPHRASE ?? '',
+      basePath: process.env.ISLYKILL_SERVICE_BASEPATH ?? '',
     },
   },
 
   contentful: {
-    accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
+    accessToken: process.env.CONTENTFUL_ACCESS_TOKEN ?? '',
   },
-} as Environment
+}
 
-const prodConfig = {
+// TODO: Remove empty-string defaulting
+const prodConfig: Environment<
+  | 'attachmentBucket'
+  | 'baseApiUrl'
+  | 'clientLocationOrigin'
+  | 'email'
+  | 'emailOptions'
+  | 'generalPetition'
+  | 'islykill'
+  | 'jwtSecret'
+  | 'presignBucket'
+  | 'userProfile'
+  | 'xRoadBasePathWithEnv'
+  | 'smsOptions'
```

Comment:
this is used in at least two places, should this be it's own type?

Reply:
This is used in two placed, both in this file, a new type has been created